src/GitHub.App/Api/ApiClient.cs

```diff
@@ -62,6 +67,60 @@ public IObservable<User> GetUser()
             return gitHubClient.User.Current();
         }
 
+        public IObservable<string[]> GetScopes()
+        {
+            return GetScopesInternal().ToObservable();
+        }
+
+        async Task<string[]> GetScopesInternal()
+        {
+            // If auth type is Basic we might be able to read /api/authorizations to get the 
+            // current scopes. However this request sometimes gets mysteriously converted to an
+            // OAuth request so if that doesn't work try reading from / and checking for the 
+            // X-OAuth-Scopes header.
+            var connection = gitHubClient.Connection;
```

Comment:
There's no point in doing this, we never have access to the actual user/password once login is completed. The password gets swapped with a token at that point.